packages/translations/locales/fr/settings.json

```
@@ -127,6 +127,112 @@
+      "customSchedule": "Remplacement/Suppléance",
+      "customScheduleDesc": "Disponible pour du travail de remplacement avec des horaires spécifiques"
+    },
+    "weeklySchedule": {
+      "title": "Horaire hebdomadaire",
+      "subtitle": "Définissez votre disponibilité régulière pour chaque jour",
+      "days": {
+        "sunday": "Dimanche",
+        "monday": "Lundi",
+        "tuesday": "Mardi",
+        "wednesday": "Mercredi",
+        "thursday": "Jeudi",
```

Biome (2.1.2)

[error] The key days was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)